docs/ske/10-integrations/10-backstage/03-generator.mdx

```diff
@@ -270,3 +285,151 @@ Resource object itself:
 * `ske.backstage.component/consumesApis`: comma-separated list of values for `spec.consumesApis`
 * `ske.backstage.component/dependsOn`: comma-separated list of values for `spec.dependsOn`
 
+## SKE Backstage Component Promise
+
+The SKE Backstage Component Promise provides additional functionality to the SKE Backstage
+integration by offloading management of Backstage Components and Templates from your
+Promises.
+
+The Promise can be found in the [Syntasso Enterprise
+Releases](http://syntasso-enterprise-releases.s3-website.eu-west-2.amazonaws.com/#promises/),
+and installed via `kubectl`:
+
+```yaml
+kubectl apply -f ske-backstage-component-promise.yaml
+```
+
+### Component updates with multiple Pipelines
+
+If your Resource workflow contains multiple Pipelines, you may wish to update the Resource
+Component in Backstage at the end of each Pipeline. For example, the Resource status could
+be updated in every Pipeline to convey the latest information about the Resource to the
+requester in Backstage.
+
+To do this, you can configure the SKE Backstage Generator to send a request to the SKE
+Backstage Component Promise in the Resource workflow by adding the `--make-request` arg to
+the SKE Backstage Generator in every Pipeline in your Resource Configure workflow, along
```

Comment:
it needs to be in promise too? to enable single file

Reply:
ah its mentioned below, maybe for sake of simplicity lets just recommend doing `--make-request` in promise workflow as well, even if its just a single pipeline? I can imagine people slipping up on this